server/server.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-undef */
+const supertest = require('supertest');
+const app = require('./server');
+
+const request = supertest(app);
+
+describe('Server endpoint', () => {
+  it('Successfully handles a get request', async (done) => {
```

Comment:
I was wondering about this as well when writing out the tests. Would it be best practice to have both this test and the next in the same `it` statement? Since its duplicate code to be calling the request.get twice.

Reply:
Yeah I'm not sure. It does repeat, but the tests are also looking for different things, so it might be best to leave them separate.